spring-integration-stomp/src/main/java/org/springframework/integration/stomp/outbound/StompMessageHandler.java

```diff
@@ -84,6 +96,10 @@ public void setHeaderMapper(HeaderMapper<StompHeaders> headerMapper) {
 		this.headerMapper = headerMapper;
 	}
 
+	public void setConnectTimeout(long connectTimeout) {
```

Comment:
I wonder if we could reuse that `recoveryInterval` from the `StompSessionManager`. Just because both these options are dancing around the same `connect` logic.